docker/conf/livy-env.sh

```diff
@@ -0,0 +1,17 @@
+# LIVY ENVIRONMENT VARIABLES
```

Comment:
are the conf files in docker/conf being used considering the individual image directories also have conf directories of their own? can we delete this docker/conf directory?